proposals/authorization-ucr/index.bs

```diff
@@ -1328,9 +1328,29 @@ thus if it can be, for example, liked, bookmarked, archived by everyone.
 * [[#req-effective-modes]]
 * [[#req-read]]
 * [[#req-write]]
- 
+
 </div>
 
+## Delegation ## {#uc-delegation}
+
+### Delegating subset of received access to another agent ### {#uc-delegation-subset}
+
+An organization *ACME*, grants Alice read-write access to one of their projects.
+Alice wants to get opinion from Bob who doesn't have any access granted directly by *ACME*.
+Alice needs to enable Bob to have read access to that project of *ACME*. She could:
+* Save the information as files and share them with Bob
+* Automate taking snapshots and sharing of the latest version with Bob each time the project gets updated
+* Setup a proxy which would allow Bob to have specific limited access impersonating Alice
```

Comment:
```suggestion
* Automate taking snapshots and sharing the latest version with Bob each time the project gets updated
* Set up a proxy which would allow Bob to have specific limited access impersonating Alice
```